ups/cell_coadds.table

```diff
@@ -1,5 +1,6 @@
 setupRequired(base)
 setupRequired(meas_algorithms)
+setupRequired(pipe_tasks)
```

Comment:
This is currently required because this repo also provides the generic coadd builder task in `_cell_coadd_builder.py`. I am guessing that we will soon be removing it out of the repo, and this dependency can be removed then.